src/components/pages/home/elements/HomePhoto.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import getPhotos from '../../../../api/photoApiClient';
+
+const defaultSrc = 'https://mars.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/02540/opgs/edr/fcam/FLB_622990123EDR_F0763002FHAZ00341M_.JPG';
+
+export default class HomePhoto extends Component {
+  constructor() {
+    super();
+    this.state = {
+      imgSrc: null,
+      date: null,
+    };
+  }
+
+  componentDidMount() {
+    getPhotos().then(response => {
+      this.setState({
+        imgSrc: response[0].cameras[0].photos[0].imgSrc,
+        date: response[0].cameras[0].photos[0].earthDate,
+      });
+    });

```

Comment:
It might read nice to assign the homepage photo to a variable, i.e.
```
const homePagePhoto = response[0].cameras[0].photos[0];
this.setState({
    imgSrc: homePagePhoto.imgSrc,
    date: homePagePhoto.earthDate,
});

Reply:
We might want to put a `.catch()` in here to make sure we don't break the page if the API doesn't find any images for this camera.